### DISCUSSION TITLE
isp.netscape.com

### DISCUSSION THREAD

**Discussion by theycallmemaki** [catppuccin/catppuccin#1841] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Maybe

**Additional context 📷 (optional)**
Dear Catppuccin team, I love your colors and use them wherever I can.
However, there is no user css for https://isp.netscape.com/ yet, which breaks the immersion for me.

I'm sure many others are missing this port too.

Please fix

**Comment by NeonGamerBot-QK** [comment]
ima start this

**Comment by NeonGamerBot-QK** [comment]
https://github.com/catppuccin/userstyles/pull/1235

**Comment by sgoudham** [comment]
Closing as future discussion should take place in the above PR.
